app/model/content/JsdToContentOptions.scala

```diff
@@ -251,5 +251,10 @@ object JsdToContentOptions {
     case _: JsdPtaSimpleAssessment => ContentOptions(
       isWelshSupported = IsWelshSupported.yes,
       title = BannerTitle(englishValue = "Pay your Simple Assessment", welshValue = Some("Talu eich Asesiad Syml")))
+
+    //TODO
```

Comment:
this has to be address during JSR work or during decoupling survey from pay-frontend